astro/src/content/blog/fusionauth-reinvent-2024.mdx

```diff
@@ -0,0 +1,64 @@
+---
+publish_date: 2024-11-15 
+title: "FusionAuth at AWS re:Invent 2024"
+description: Join us at AWS re:Invent 2024. Meet the team, get demos, and win some great prizes.
+authors: Dan Stotts
+image: /img/blogs/reinvent-2024/re-invent-2024.png
+categories: Events
+tags: reinvent, 2024 events, fusionauth
+excerpt_separator: "{/* more */}"
+---
+
+We’re thrilled to announce that FusionAuth will be at [AWS re:Invent 2024](https://reinvent.awsevents.com/), and we can’t wait to connect with you! We’ve planned an array of activities and opportunities for engagement throughout the event.
+
+## Ways to Engage:
+
+**1. Find Us at Our Booth**
+
+Swing by [Booth #1079](https://awsreinvent24.mapyourshow.com/8_0/exhview/index.cfm?orsearchtype0=exhibitor&orsearchvalue0=1112&orsearchdisplay0=Exhibitor&orsearchvaluedisplay0=FusionAuth) during Expo hours to meet our team, see a demo, and learn how FusionAuth can supercharge your authentication and authorization needs.
+
+**Booth Hours (PST):**
+
+- **Monday, December 2**: 4 PM - 7 PM (Welcome Reception)
+- **Tues & Weds, Dec. 3&4**: 10 AM - 6 PM
+- **Thursday, Dec. 5**: 10 AM - 5 PM
+
+**Daily Prizes: Enter & Win!**
+
+Stop by our booth and chat with a FusionAuth representative to enter our daily drawing for a chance to win amazing prizes like:
+
+- Nintendo Switch
+- Keychron Keyboard
+- LEGO Sets
+
+Winners will be drawn daily. Prizes awarded Monday through Thursday. You must be present to win!
+
+**2. Book Time with an Expert**
+
+Got questions about authentication and authorization? Need some best practices or just want to swap stories? Our experts are here for you. Plus, they might even throw in a few Chuck Norris jokes.
```

Comment:
Are we leaning into this? Is this a free support session? What is this going to offer beyond a demo? Do you want to drop the fact that it will be with the CTO or senior member of the engineering team (if that is true)? How can you make this something someone will actually want to do?

How long is it?

Reply:
I shot this over to Dan S for clarification.